components/EventItem.tsx

```diff
@@ -1,33 +1,29 @@
 import React from "react"
 import { Image, StyleSheet, Text, View } from "react-native"
 import { Event } from "@lib/events/Event"
-import IconButton from "./common/IconButton"
-import { Divider, Icon } from "react-native-elements"
+import { Divider } from "react-native-elements"
 import { Shadow } from "react-native-shadow-2"
 import tinycolor from "tinycolor2"
 import { placemarkToFormattedAddress } from "@lib/location"
 import { compactFormatMiles } from "@lib/DistanceFormatting"
+import { MaterialCommunityIcon, MaterialIcon } from "./common/Icons"
 
 interface Props {
   event: Event
 }
 
 const EventItem = ({ event }: Props) => {
-  const numAttendees = 1
-  const distance = 0.5
-  const shadowColor = "#bdbdbd"
-  const lightEventColor = tinycolor(event.color).lighten(10).toString()
-
-  const onPressMore = () => {
-    return null
-  }
+  const numAttendees = 1 // Eventually will come from event object
+  const distance = 0.5 // Eventually will be calculated from the location given in the event object
+  const shadowColor = "#ebebeb"
+  const lightEventColor = tinycolor(event.color).lighten(25).toString()
 
   return (
     <Shadow
-      distance={5}
+      distance={18}
```

Comment:
We may want to store this shadow in a variable to be used anywhere we need a shadow, for consistency across the app